docs/src/main/tut/client.md

```diff
@@ -30,6 +30,15 @@ A good default choice is the `PooledHttp1Client`.  As the name
 implies, the `PooledHttp1Client` maintains a connection pool and
 speaks HTTP 1.x.
 
+First, we'll need a dependency on the http4s blaze-client
```

Comment:
Looks like I commented on the wrong ticket...

Are we assuming people know by this point to put it in build.sbt?